controllers/users.js

```diff
@@ -495,6 +495,11 @@ const filterUsers = async (req, res) => {
   }
 };
 
+const usersForInDiscord = async (req, res) => {
```

Comment:
can we change it to `nonVerifiedDiscordUsers`?

Reply:
done